### PR TITLE
BP-2/chore/install packages

### PR DESCRIPTION
**Pull request by jay1805**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -258,3 +258,5 @@ $RECYCLE.BIN/
 
 # Windows shortcuts
 *.lnk
+
+target/
```
Comment (omjadas) [inline .gitignore:262]: add a trailing newline

File: preprocessing-plugin/src/main/webapp/WEB-INF/applicationContext.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="
+        http://www.springframework.org/schema/beans     
+        http://www.springframework.org/schema/beans/spring-beans-3.0.xsd
+        http://www.springframework.org/schema/context 
+        http://www.springframework.org/schema/context/spring-context-3.0.xsd">
+ 
+ 	<context:component-scan base-package="org.apromore.plugin" />
+ 
+</beans>
```
Comment (omjadas) [inline preprocessing-plugin/src/main/webapp/WEB-INF/applicationContext.xml:12]: add a trailing newline

File: preprocessing-plugin/src/main/webapp/index.zul [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<zk>
+	<window apply="org.zkoss.bind.BindComposer"
+		viewModel="@id('vm')@init('org.apromore.plugin.MyViewModel')"
+		validationMessages="@id('vmsgs')">
+		<hlayout>today is <label value="@load(vm.answer)" /></hlayout>
+		<button label="ask" onClick="@command('cmd')"/>
+	</window>
+</zk>
```
Comment (omjadas) [inline preprocessing-plugin/src/main/webapp/index.zul:8]: add a trailing newline

**Comment by omjadas** [comment]
we also need junit for testing

**Comment by jay1805** [comment]
Fixed all the above @omjadas

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+							<descriptors>
+								<descriptor>src/main/assembly/webapp.xml</descriptor>
+							</descriptors>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+		</plugins>
+	</build>
+</project>
```
Comment (omjadas) [inline pom.xml:173]: one more newline

**Pull request merged**